docs/guides/protect-page-login/go.mdx

```diff
@@ -59,13 +65,13 @@ import CodeBlock from '@theme/CodeBlock'
 
 ## Validate and Login
 
-Next we will create a middleware which will check with your Ory project if the
-user has a valid session. Notice here that we're taking the current `request`
-cookies and passing them along to the Ory client.
+Next, create middleware that checks if the user has a valid session with the defined Ory project. The session is considered valid when the user is signed in to an account created in the defined project.
```

Comment:
It needs a bit of rewording because project provides you services with capabilities such as log in, registration etc that you consume later in your application, also we provide out-of-the-box managed UI as a demo application implementing those services. So it's not signing in to the project strictly.  Users log in to the application consuming our services for authentication. In this tutorial, we are using this managed UI demo application as a reference implementation.

Reply:
removed the "sign in to Ory project" wording